oss_src/unity/python/sframe/data_structures/sframe.py

```diff
@@ -1641,6 +1631,36 @@ def read_json(cls,
                 return g
         else:
             raise ValueError("Invalid value for orient parameter (" + str(orient) + ")")
+    
+    @classmethod
+    def set_spark_integration_jar_path(cls, jar_path):
+        """
+        Set the absolute path of the jar file required to enable GraphLab Create's
+        integration with Apache Spark.
+        """
+        global SFRAME_GRAPHLABUTIL_REF
+        if not isinstance(jar_path, str):
+            raise TypeError("The jar path must be str type")
+        if not os.path.isfile(jar_path):
+            raise RuntimeError("Cannot find the file %s" % jar_path)
+        if not jar_path.endswith(".jar"):
+            raise RuntimeError("The jar path must point to a jar file")
+        if SFRAME_GRAPHLABUTIL_REF is not None:
+            raise RuntimeError("The jar path should be set before calling to_rdd(), from_rdd(), or to_spark_dataframe() methods.")
```

Comment:
Is this the correct error message?  I'd be pretty confused, as I'd think this is how to set the jar path.